ui/app/components/clients/history.js

```diff
@@ -38,10 +38,15 @@ export default class History extends Component {
   years = Array.from({ length: 5 }, (item, i) => {
     return new Date().getFullYear() - i;
   });
+  currentDate = new Date();
+  currentYear = this.currentDate.getFullYear(); // integer of year
+  currentMonth = this.currentDate.getMonth(); // index of month
 
   @tracked isEditStartMonthOpen = false;
   @tracked startMonth = null;
   @tracked startYear = null;
+  @tracked allowedMonthMax = 12;
```

Comment:
Curious about this. The name would imply they can maybe only return 12 months of data. However, some licenses go two years. Not sure if that's how it's being used, but thought it worth mentioning.